extensions/blocks/podcast-player/templates/playlist-track.php

```diff
@@ -30,16 +32,27 @@
 
 <li
 	class="<?php echo esc_attr( trim( $class ) ); ?>"
+	[class]="'<?php echo esc_attr( trim( 'jetpack-podcast-player__track ' . $secondary_colors['class'] ) ); ?> ' + (currentTrack == <?php echo esc_attr( $index ); ?> ? 'is-active' : '')"
 	style="<?php echo esc_attr( $style ); ?>"
+	option="<?php echo esc_attr( $index ); ?>"
+	<?php echo $is_active ? 'selected' : ''; ?>
```

Comment:
I would still `esc_attr()` here. I was told during my trial that even with hardcoded values you should always wrap in an escape function as it ensures that if someone comes along and extracts the value to a variable they won't forget to escape it.